src/sentry/preprod/tasks.py

```diff
@@ -277,13 +278,22 @@ def _assemble_preprod_artifact_size_analysis(
             pass  # Ignore cleanup errors
         raise Exception(f"PreprodArtifact with id {artifact_id} does not exist")
 
+    size_analysis_results = SizeAnalysisResults.parse_raw(assemble_result.bundle_temp_file.read())
```

Comment:
Technically `parse_raw` is deprecated in favor of `model_validate_json`, but the pydantic version used by the BE is quite old, so we're stuck with this for now.